[rdu-2024] Create Raleigh 2024 event

jrn00 opened the pull request:
Copied Raliegh 2023 files over to Raleigh 2024

Removed all 2023 specific data

changed 2023 dates to 2024 dates

Created place holder files within some data folders.

jrn00 commented:
This PR replaces PR#13288

jyee commented on data/events/2024-raleigh.yml:
```diff
@@ -0,0 +1,108 @@
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
```
I noticed that you didnt copy the `/static/events/2023-raleigh` folder into a new 2024 folder. In particular, you'll probably want the `logo-square.jpg` file and the `organizers` subfolder. Currently the contact page for this PR will list the organizers, but their photos are broken.

Generally we recommend not copying old events because things like this get missed (also you might miss updates that have been added since older event files dont have them). Take a look at the `utilities` folder. The README there has info about scripts you can run that make things a bit easier.

jyee commented on content/events/2024-raleigh/sponsor.md:
```diff
@@ -0,0 +1,68 @@
+
+We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at [{{< email_organizers >}}].
+
+For more information, please view our full prospectus. Sponsor package details are here : [Sponsorship Package](https://assets.devopsdays.org/events/2023/raleigh/devopsdaysraleigh2023sponsorshippacket.pdf)
```
Looks like this is referencing this year's prospectus. It's probably best to remove this link to avoid confusion, until you have one for 2024 to share.

jyee commented on data/events/2024-raleigh.yml:
```diff
@@ -0,0 +1,108 @@
+  - name: location
+  - name: registration
+    url: "https://www.eventbrite.com/e/devopsdays-raleigh-2024-tickets-615268222817?aff=ebdsoporgprofile"
+  - name: program
```
Since you dont have speakers or a program yet, you can comment these out so the links don't appear in the navigation.

jyee commented on data/events/2024-raleigh.yml:
```diff
@@ -0,0 +1,108 @@
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2023-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2024-04-10  # The start date of your event. Leave blank if you don't have a venue reserved yet.
```
Since we've had issues with timezone weirdness, can you update these to follow the new format (see yaml comments above). Basically you'll want something like `2024-04-10T08:00:00-04:00` (EDT time is GMT-4)

jyee reviewed (changes requested):
Mostly minor changes that are easy to fix.

jrn00 commented:
Thanks Jason, I'll take care of it.

jrn00 commented:
@jyee, I made the requested changes. Thank you for showing me those. In the future, I'll utilize the scripts. To be honest, I was doing alot of this through the GitHub app on my phone while I was at a baseball game :)

jyee commented on data/events/2024-raleigh.yml:
```diff
@@ -0,0 +1,108 @@
+    gitlab: "n3ff"
+    github: "JRN00"
+    twitter: "GitDevOps"
+    image: "jeremy-neff.png"
```
image here should be `.jpg` and not `.png`

jyee commented on content/events/2024-raleigh/location.md:
```diff
@@ -0,0 +1,24 @@
+{{< event_map >}}
+
+<br>
+If you are traveling to attend this event, the McKimmon Conference & Training Center is approximately a 25-minute drive from the RDU Airport via I-40 (East). For a list of hotels and their distance to the vicinity of the McKimmon Conference & Training Center for your convenience: <a href="https://assets.devopsdays.org/events/2023/raleigh/Hotels_DevOpsDays_Raleigh_2023.pdf"><strong>click here</str ...
```
Is the hotel guide here still valid? Probably ok to leave if it is or maybe adjust to something like "For hotel recommendations, see our guide from 2023 (note that prices will have changed)"

jyee reviewed (changes requested):
Two super minor changes.

jrn00 commented:
@jyee. Corrected picture extension. I removed the link to the hotels for the time being. I'll update the assets page once I get the newer prices.

Thanks again for QC'ing me!
